### PR TITLE
docs and log fixes

### PR DESCRIPTION
**Pull request by praveen-garg**

### CODE REVIEW

**Comment by praveen-garg** [comment]
@MarkRiddoch Could you please set the default branch as develop; and make develop and master protected.

**Review by ashish-jabble — APPROVED** [review]

**Review by YashTatkondawar — APPROVED** [review]

File: docs/index.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
++===========+================================================================+
+| timestamp | The timestamp will be generated locally by the plugin.         |
++-----------+----------------------------------------------------------------+
+| asset     | The name of the asset configured.                              |
++-----------+----------------------------------------------------------------+
+| readings  | A JSON object parsed from received message payload.            |
++-----------+----------------------------------------------------------------+
+
+
+Example of Published Message Payload
```
Comment (MarkRiddoch) [inline docs/index.rst]: This example des not seem to correspond to the payload description above

**Review by MarkRiddoch — CHANGES_REQUESTED** [review]
It looks to me like the documentation is not consistent

File: docs/index.rst (hunk 1 above)
Comment (praveen-garg) [reply docs/index.rst]: Sorry, I'm not following! The published message payload will be stored in `readings: {"humidity": 93.29, "temp": 16.82}`

Do you mean, I should change the heading to `*Stored Readings Object Structure*`
and move the (re-phrased) line

> The content of the *readings* object is a set of JSON properties. The payload (byte-array) of published messages should be a JSON object. 

The content of the *readings* property is a set of JSON object properties i.e. the payload (byte-array) of published message.

after the table?

**Review by MarkRiddoch — APPROVED** [review]
Thanks

**Pull request merged**